bundles/fish.payara.eclipse.tools.server/plugin.xml

```diff
@@ -922,6 +922,12 @@
             id="fish.payara.eclipse.tools.server.commands.OpenDomainHomeCommand"
             name="OpenDomainHomeCommand">
         </command>
+        <command
+			id="myplugin.newMenuItem"
+			name="New Menu Item"
+			description="This is a new menu item"
```

Comment:
Name and description are default that should be updated specific to feature.

Reply:
Command removed!